shiny/experimental/ui/_sidebar.py

```diff
@@ -144,13 +137,14 @@ def layout_sidebar(
     )
 
     res = div(
-        sidebar_dependency(),
-        sidebar_js_init(),
         {"class": "bslib-sidebar-layout"},
         {"class": "sidebar-right"} if right else None,
```

Comment:
```suggestion
        {"class": "sidebar-right"} if  sidebar.position == "right" else None,
```

And remove `right` definition above.